wasmcloud_host/lib/wasmcloud_host_web/templates/layout/root.html.leex

```diff
@@ -89,7 +89,7 @@
               <div class="c-avatar"><img class="c-avatar-img" src="<%= Routes.static_path(@conn, "/images/wasmcloud_extraspace.png") %>" alt="wasmCloud logo"></div>
             </a>
             <div class="dropdown-menu dropdown-menu-right pt-0"><a class="dropdown-item" href="#">
-                Version<span class="badge badge-success mfs-auto">0.19.0</span></a><a class="dropdown-item" href="#">
+                Version<span class="badge badge-success mfs-auto">0.50.0</span></a><a class="dropdown-item" href="#">
```

Comment:
Is there any way we can get this version information programmatically?

Reply:
Addressed in later PR comment (TL;DR, yes)